examples/webmcp-challenge/styles.css

```
@@ -0,0 +1,1652 @@
+  align-items: center;
+  gap: 8px;
+}
+
+.shortcut-button {
+  display: inline-flex;
+  align-items: center;
+  gap: 7px;
+}
+
+.shortcut-button kbd {
+  border-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)